src/service/account.ts

```diff
@@ -0,0 +1,35 @@
+import type { SecretReferenceResolver } from "./types";
+import { Client, createClient } from "@1password/sdk";
+import process from "node:process";
+import { envServiceAccountToken } from "../constants";
+import assert from "node:assert";
+import { version } from "../../package.json";
+
+export class Account implements SecretReferenceResolver {
```

Comment:
`[suggestion]`

Consider renaming this class to `ServiceAccount` since simply `Account` is a bit confusing and doesn't suggest that we authenticate the action using a service account.

Reply:
solved by 45ee2c20c5a769c79f653b4235273ccc96689422